code/game/objects/items/devices/anomaly_neutralizer.dm

```diff
@@ -0,0 +1,20 @@
+/obj/item/anomaly_neutralizer
+	name = "anomaly neutralizer"
+	desc = "A one use device capable of instantly neutralizing anomalies."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "memorizer2"
+	item_state = "electronic"
+	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = ITEM_SLOT_BELT
+	flags_1 = NOBLUDGEON_1
+
+/obj/item/anomaly_neutralizer/afterattack(atom/target, mob/user, proximity)
+	..()
+	if(!proximity || !target)
+		return
+	if(istype(target, /obj/effect/anomaly))
+		var/obj/effect/anomaly/A = target
+		to_chat(user, "<span class='notice'>[src] crumbles to dust from the strain of neutralizing [A]!</span>")
```

Comment:
Crumbling to dust is something I'd expect from a magic item, not technology.
Maybe re-fluff it to saying it explodes or something.

Reply:
Fair

Reply:
> crumbles to dust
> isn't deleting

???

Reply:
its a beta